%YAML:1.0
---
# Primary blaser camera (with Magsafe)
model_type: MEI
camera_name: camera
image_width: 2448
image_height: 1840
mirror_parameters:
   xi: 1.8516756874116116e+00
distortion_parameters:
   k1: -8.1988658236129097e-01
   k2: 6.3879257711668663e-02
   p1: 8.5244705461647498e-03
   p2: -8.1949134295897057e-03
projection_parameters:
   gamma1: 3.3459441827828582e+03
   gamma2: 3.3406052649685403e+03
   u0: 1.2074996866814947e+03
   v0: 8.7172321297820872e+02
width: 2448
height: 1840

# checkerboard parameters
target_rows: 7
target_cols: 10
square_size: 0.01 # meters

# laser detection parameters for HSV thresholding
# For red laser that 
hue_min: 0
hue_max: 30
sat_min: 0
val_min: 100 # brightness
val_ratio: 0.05 # val_thresh = max(val_min, image_max_val * val_ratio)


segment_length_min: 15
vis_laser_detection: 0
laser_horizontal: 1
